.github/workflow-template/jobs/e2e-source.yml

```diff
@@ -94,11 +94,6 @@ jobs:
           chmod +x ./target/debug/risedev-playground
           chmod +x ./target/debug/risingwave_regress_test
 
-      - name: Run batch tests on Postgresql
-        timeout-minutes: 10
-        run: |
-          sqllogictest -p ${{ env.PG_PORT }} './e2e_test/batch/**/*.slt'
-
```

Comment:
Do not know why e2e-source has a test on PG. Looks unnecessary so delete. cc @shanicky @tabVersion

Reply:
This is used by regression test?

Reply:
cc @liurenjie1024

Reply:
It's to make ensure that we have the same query result as pg, I think it's necessary.

Reply:
No. If regression tests, at least RW should also run it. However RW is not.

Reply:
No, it's not used by regression test.